west.yml

```
@@ -42,7 +42,8 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 31529f68ec7fb266a4ec1761009e100ef9850ea5
+      revision: 1847721da924e6819df30248105d83a88a15285c
```

> I need to wait for fw-nrfconnect-zephyr PR 158, and then update west.yml before merge.

@thst-nordic To allow CI to automatically build and also make it visible to reviewers that this PR is dependent on another PR, then you can use the following syntex instead of specifying the SHA.
Then you also avoid the need of updating the SHA every time you push to PR#158.
```suggestion
      revision: pull/158/head
```
and then update to the final SHA when https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/158 is merged.